src/development/inter-process-communication.md

```diff
@@ -282,7 +283,7 @@ error type that implements `serde::Serialize`. <br> In the following example, we
 use a crate called [`thiserror`] to help create the error type. It allows you to
 turn enums into error types by deriving the `thiserror::Error` trait. You can
 consult its documentation for more details.
-
+<!-- TODO: in the codesnippet below, consider mentioning that the custom serialize implementation is not needed if the from macro isn't used (or if from is implemented manually). Maybe this would make it too complicated -->
```

Comment:
Is that actually true? If you use `#[from]` on all fields you can derive serialize?

Reply:
Ahhhh yeah makes sense! But we shouldn't mention that probably 🤔 a custom macro impl or copious use of `map_err` are not as clean as a simple Serialize implementation